templates/zerver/help/dark-theme.md

```diff
@@ -16,6 +16,6 @@ for working in a dark space.
 The default is **Sync with computer**, which detects which theme to use based
 on the color scheme used by your operating system.
 
-You can also specify **Night mode** or **Day mode** if you'd like
+You can also specify **Dark theme** or **Day mode** if you'd like
```

Comment:
Do we also want to rename `Day mode`. We inconsistently use `Light mode` and `Day mode`. I think we can use `Light theme`.